src/utils/computed.js

```diff
@@ -76,7 +88,14 @@ module.exports = function (o) {
   if (isFunction(o)) {
     o = {
       watch: arguments[1],
-      get: o
+      get: o,
```

Comment:
can be fixed by 
```js
  // before es6
  if (isFunction(o)) {
    o = {
      watch: Array.prototype.slice.call(arguments, 1),
      get: o
    }
  }

  // or only support node >= 6
module.exports = function (o, ...args) {
  if (isFunction(o)) {
    o = {
      watch: args,
      get: o
    }
  }


}
```

Reply:
Thanks and one question, base on your solution, what is your idea if I want a new `option` property?

Reply:
oh, according your title description, I didn't notice that. :sweat_smile: 
what options of `computed` do you want to support ? 
I think if you want to add more properties in computed type, should only use METHOD 2.
```
**METHOD 2:**
@method computed
@param {Object} object The object containing the [set/get/watch/description...] property.
```

Reply:
I am still prefer my approach, but I am okay with yours, let's wait other reviewers' inputs